packages/connectors/connector-apple/src/index.ts

```diff
@@ -13,7 +13,7 @@ import {
   validateConfig,
   ConnectorType,
 } from '@logto/connector-kit';
-import { generateStandardId } from '@logto/core-kit';
+import { generateStandardId } from '@logto/shared/universal';
```

Comment:
nit: could directly import from `@logto/shared`? feels like the connector code is referenced by the core, and runs on node env only.